src/components/Table/TableFormatters/AmountFormatter.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <DetailFormatter :col="col" :row="row" :prevant-click="formatterArgs.preventClick">
+  <DetailFormatter :col="col" :row="row" :prevent-click="formatterArgs.preventClick">
     <template>
       <el-popover
         :disabled="!showItems"
```

Comment:
These components are not implemented and don't contain functional logic.

It looks like the template tag is missing at the end of the code.

If you want to know about any discrepancies in code style between two versions or discuss possible improvements/suggestions on this file, please share more context around your request.